scripts/ashell-monkey.py

```diff
@@ -0,0 +1,99 @@
+# This tool can test many JS test files at once using ashell. To use, flash a
```

Comment:
How about add:
`#!/usr/bin/env python3`

... and copyright below that.

... and you'll have to fix your print statements to use parentheses, required in Python3, hopefully nothing else.